test/e2e/performanceprofile/testdata/must-gather/must-gather.e2e/quay-io-openshift-kni-performance-addon-operator-must-gather-sha256-e0807f2ff1ff1754cdb7475e665c6c60fb1faa1f14eb0f2f7a791388ac8b33e8/cluster-scoped-resources/machineconfiguration.openshift.io/kubeletconfigs/performance-performance.yaml

```diff
@@ -132,7 +132,10 @@ spec:
       full-pcpus-only: "true"
     cpuManagerReconcilePeriod: 5s
     evictionHard:
+	    nodefs.available: 15%
```

Comment:
weird spacing here, let's be consistent with the original indent

Reply:
right, some error with IDE tabs